code/obj/item/satchel.dm

```diff
@@ -8,7 +8,10 @@
 	w_class = W_CLASS_TINY
 	event_handler_flags = USE_FLUID_ENTER | NO_MOUSEDROP_QOL
 	var/maxitems = 50
+	var/max_stack_scoop = 20 //! if you try to put stacks inside the item, this one limits how much you can in one action. Creating 100 items out of a stack in a single action should not happen.
 	var/list/allowed = list(/obj/item/)
+	var/list/exceptions = list() //! this list are for items that are in the allowed-list for other reasons, but should not be able to be put in satchels
```

Comment:
please initialize these in new()
https://hackmd.io/@goonstation/code#StartupRuntime-trade-offs-with-lists-and-the-%E2%80%9Chidden%E2%80%9D-init-proc